### ISSUE TITLE
resizableLabels() shortening to SA unlike for CT

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1367]
Something is funny in `resizableLabels()`, as attached. It abbreviated sal but not temp, even though I reckon
```
Absolute Salinity [g/kg]
```
is shorter than
```
Conservative Temperature [dC]
```

![screen shot 2018-01-19 at 12 47 20 pm](https://user-images.githubusercontent.com/99469/35161793-a3046e3c-fd17-11e7-903a-958175c7f691.png)

**Comment by dankelley** [comment]
Oh, actually this is not about `resizableLabel`, since the following also displays the error
```R
library(oce)
data("ctd")
plot(ctd, which="SA")
```

**Comment by dankelley** [comment]
This was actually in `plot,ctd-method`; fixed now in "develop" commit d9f63cced23b8cf9068ad687b18c316563c35e24
